tools/validate_config.py

```
@@ -0,0 +1,324 @@
+        logging.getLogger().setLevel(logging.DEBUG)
+    
+    if not args.config_file.exists():
+        logger.error(f"❌ Fichier introuvable : {args.config_file}")
+        return 1
+    
+    # Chargement de la configuration
+    try:
+        with open(args.config_file, 'r', encoding='utf-8') as f:
+            config = json.load(f)
+    except json.JSONDecodeError as e:
+        logger.error(f"❌ Erreur JSON : {e}")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)